src/global.h

```diff
@@ -195,6 +199,9 @@ LED bar:      lbr
 // list
 #define PING_UPDATE_TIME_SERVER_LIST_MS 2000 // ms
 
+// defines the interval between Channel Level updates from the server
+#define CHANNEL_LEVEL_UPDATE_INTERVAL   100  // number of frames
```

Comment:
The number is not given in ms but in frames. So it is dependent on the frame size and sample rate, right?

Reply:
Yes -- will your work affect this?  If the frame frequency drops, I guess the channel level update interval should also drop?

Reply:
I am currently working on supporting different frame sizes in the server. So I have to make sure that your update time stays the same no matter which frame size is in use by the server right now. No big deal, I just have to remember it that there is something to do :-).